### DISCUSSION TITLE
Prevent unloading certain tabs based on the domain

### DISCUSSION THREAD

**Discussion by LiquidFiend** [zen-browser/desktop#1970] [Ideas]
Hi there,

I would like to prevent unloading tabs for mail.google.com and calendar.google.com so that I can get notifications for my email and calendar without fear of them being unloaded.

I can manually select them and mark them as 'Prevent Unload' but it would be nice to be able to have it saved in the settings so I don't need to think about it.

Thank you!

**Comment by mikedlr** [comment]
For that feature as you described, which is really useful for the [idea of Zen as a web communicator](https://www.reddit.com/r/zen_browser/comments/1ge7sw5/use_of_zen_as_a_web_communicator_mostly_working/) don't you only really need one tab to stay open? Maybe this should be a feature of pinned tabs as well / instead?

For notifications from some sites it seems to be possible to subscribe to push notifications even when there's no actual page open. Is that something that could work better than keeping the page open here?

**Reply by LiquidFiend** [reply]
It sounds like that would be overkill for what I need, but it does seem nice.

I ended up using a solution from this reddit post: https://www.reddit.com/r/zen_browser/comments/1g0ntdi/comment/lrrhyqr/

Go to about:config, search for zen.tab-unloaded and then put "mail.google.com,calendar.google.com" into the excluded URLs.

It's nice that this is available, but ideally it would be easier to work with.
